installing/installing_azure/ipi/installing-azure-vnet.adoc

```diff
@@ -37,7 +37,13 @@ include::modules/installation-azure-arm-tested-machine-types.adoc[leveloffset=+2
 include::modules/installation-azure-trusted-launch.adoc[leveloffset=+2]
 include::modules/installation-azure-confidential-vms.adoc[leveloffset=+2]
 
-include::modules/installation-azure-config-yaml.adoc[leveloffset=+2]
+include::modules/installation-azure-config-yaml-simple.adoc[leveloffset=+2]
```

Comment:
Compared with 4.18 official [doc](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/installing_on_azure/installer-provisioned-infrastructure#installing-azure-vnet), there is a dedicated section - `Define the network and subnets for the VNet to install the cluster under the platform.azure field` to guide user to modify the install-config.yaml properly, but from the new version of preview doc, there are no such section, personally I think it is not very friendly to ready for users.

Reply:
Do you think that the sample install-config.yaml should contain the following parameters?

```
networkResourceGroupName: <vnet_resource_group> 
virtualNetwork: <vnet> 
controlPlaneSubnet: <control_plane_subnet> 
computeSubnet: <compute_subnet> 
```
I can add them if you think they're important, but I will not be adding descriptions to the parameters. This file is changing away from being a full description of all the parameters, towards being a simple example of what the install-config.yaml syntax looks like.

Reply:
I do not think we need to include the parameters in the sample install-config.yaml. But it is important to [Existing vnet](https://92943--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/ipi/installing-azure-vnet.html#installation-azure-config-yaml-simple_installing-azure-vnet) , currently users can not find out how to customized install-config.yaml to install clusters in an existing vnet.